articles/140_topic_and_service_name_mapping.md

```diff
@@ -243,41 +221,72 @@ Here is a non-exhaustive list of prefixes:
 | ROS Parameter        | rp     |
 | ROS Action           | ra     |
 
-While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `__`, can be considered part of the prefix.
+While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `/`, can be considered part of the prefix.
```

Comment:
Do you want me to remove that sentence or rather explain that we truncate the leading and trailing forward slashes ?